src/ray/protobuf/export_event.proto

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020-2021 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+option cc_enable_arenas = true;
+
+package ray.rpc;
+
+import "google/protobuf/struct.proto";
+
+message ExportEvent {
+  enum SourceType { EXPORT_TASK = 0; }
+
+  // event_id is the unique ID of this event
+  string event_id = 1;
```

Comment:
I don't think `event_id` is needed, basically each event log line is unique.

Reply:
Yes, each log line is a unique event but I wanted to keep the `event_id` to easily identify individual events. Otherwise we would need to identify by timestamp and source type.